src/setup/serverless-config.go

```diff
@@ -149,19 +171,35 @@ func DeployService(path string) string {
 	return slsDeployMessage
 }
 
+// DeployContainerService deploys a container service to cloud provider
+func (s *Serverless) DeployContainerService(subex *SubExperiment, index int, imageLink string, path string, region string) {
+	switch s.Provider.Name {
+	case "gcr":
+		log.Infof("Deploying container service(s) to GCR...")
+		for i := 0; i < subex.Parallelism; i++ {
+			name := createName(subex, index, i)
+
+			gcrDeployCommand := exec.Command("gcloud", "run", "deploy", name, "--image", imageLink, "--allow-unauthenticated", "--region", region)
+			deployMessage := util.RunCommandAndLog(gcrDeployCommand)
+			log.Info(deployMessage)
+			subex.Endpoints = append(subex.Endpoints, EndpointInfo{ID: GetGCREndpointID(deployMessage)})
+			subex.AddRoute("")
+		}
+	default:
+		log.Fatalf("Container deployment not supported for provider %s", s.Provider.Name)
+	}
+}
+
 // GetEndpointID scrapes the serverless deploy message for the endpoint ID
 func GetEndpointID(slsDeployMessage string) string {
```

Comment:
Should we change `GetEndpointID` to `GetAWSEndpointID`? So it is clear how the two functions are different